test/mpi_graph_io_tests.cpp

```diff
@@ -556,4 +556,70 @@ TEST_CASE("MPI, all_read_binary_graph_partition, undirected, unweighted, static,
     REQUIRE(vertex_count == enzymes_g1_vertex_count);
     REQUIRE(edge_count == enzymes_g1_edge_count);
     REQUIRE(edges.size() == edge_count);
+}
+
+TEST_CASE("MPI", "read")
+{
```

Comment:
Maybe add the graph you want to read for better error handling

Reply:
Done.